cmd/protoc-gen-cpp-tableau-loader/embed/hub.pc.h

```diff
@@ -27,6 +28,32 @@ struct LoadOptions {
   bool ignore_unknown_fields;
 };
 
+inline Format Ext2Format(const std::string& ext) {
+  if (ext == kJSONExt) {
+    return Format::kJSON;
+  }
+  if (ext == kTextExt) {
+    return Format::kText;
+  }
+  if (ext == kBinExt) {
+    return Format::kBin;
+  }
+  return Format::kUnknown;
```

Comment:
- Please use `switch case` instead.
- This function is not small, DO NOT use `inline` keyword (including func `Format2Ext` below), and put implementation code to `cpp` file.
- Also need add logic to files: *_lab/cpp/src/demo/hub.pc.{h,cpp}*

Reply:
Suggestions have been adopted except the first one for `switch case` could not applied to `std::string` directly.